packages/site/src/data/react/communities.ts

```diff
@@ -223,7 +223,8 @@ export const communities: Community<typeof communityTags[number]>[] = [
NTpNQZ9UE854lyGEd0Q+Nh8tq4+TMM6WGlN65K/VbR6m+tO8rbjyrPhZPDf5RXitiv+Fk8N/lFa668flXI0HtR/CM8NvlBS+Xf8RqfDHkNURR/CM8NvlBS+Xf8AEanwx5DVol+oj9r8YGf0vn6kDJqPIVOZbOviFQe+b5DFBP1HkUvlS+9ZMe+9EBbGvxU/8ZeMSNnYRS7V9bnYAG37So3v7xkA6HBcUuigf7zvH05v7cepaMYr9H98fMyht5M51SWT3dDfAm/QkUapDAnWnB7yb9CRb635cuT8DGOsjwiBFtZiERFACIiAtHch7nn8aPIaunyt7hqvES+QVzG5D3PP40eQ1dPlb3DVeIl8gryGM/XS+5eReh+X1PzKHVqbmuTbGRNqpWgzSaY7j4NmoOHfO132EDbeq3al+hMKAEEQb2u9x2tqtmCy7GWq0oUlCP1PTyWzvNOHinK72GzI8NBLiA0aSXEBoG0k6lr0dfDLfeZo5La96ka+3Lmkrit13fd5hzb7xnu3y2rOsN7zuLt+e3Eq1pal8b2yRPLJGm7XNNiD6uLhXOwmSliKKqZ9m76Lbt/8GydbNlaxfeL4ZFUROinaCw8PxmH5bTwEbVQ+IU4jlfG2RsjWuID2G7HganA8il8osr6mrGa8iOHReOK4a48JedbtPBq1aOFc+urkzB1cPB58tf060uv00b7mmtNSegIiLpmkIiIDLjc3OknWTrJ2rYw11p4jskjPQ8LWQHYokrpok9qt15Hna5x6SV4oilKwPWnPZt8Jv5hS+XB98KnxnotUGHW07NKmMsX3r6k/Ou6tHmWhr8eP2y8YEr5ffEh3aluYy+88h2uK019zPznE7TdbmtN/ez0IPetZZkB2xOP+Inb6IUi1/vW4fTW/tnepaeIj2ul8S791UL1z/cBH0pp/AeFVn8UY/f4SaM1rfIi19wylpuNdnDmc0tPUSvhFbtcwMrCIpICIigBERAWjuQ9zz+NHkNXT5W9w1XiJfIK5jch7nn8aPIaunyt7hqvES+QV5DGfrpfcvIvU/wAvqZRCuDc4xxs9K2En26EBhHC5g0MeNotZp4xxhU8vajq5IniSF5ZI3U5psR6xxHQV6XHYRYqnmamtKfvY1rKlOeY7n6Enha9pZI0OY4Wc1wBa4bCDrXC43uaxPu6jk3p38uS7oeQO7Zv3lr4HumDQ2ti0/wA2EaDxujOrlBPIF3WGYpDUNz6eVsjeHNPZN4nNOlp4iF5lwxeBd9K464vy8GW7wqe9JR+M4FUUrrVERaDoa8aYneC4aOY6eJRq/RNVTMkY6OVgfG4Wc1wu0ql8tMnTRz5rbmB93RE6xbtoydouNPCCONdzAZTWIeZNWl3PzvwK9WlmaVqOeREXUNIREQBERAEREBhwuLLZxKq32aSXVnvc6x1jOJIC10UWV7+/ehALKwiyBu4g72un4oXfuahGv9yub88w/hvC1pps4MHyGlvLd73+msh/YFu1zT0Bw861KFklxb72/MlnkiIthAREQBEU7kZQ089U2KrLg118wB2a17+BjnawCL6rabDhWFSapwc3s06NZKV3YioKKR7HyMjc6OMXe4NJYzUNJ5/PqWuv0PDRxtj3pkbWxWLcwNAjsdBGbq0rha/cwjdITBUmOMntHxb4W8TXZw0cvSVyqGWaU2+kWatmt9tk9Pdu1G+WHktR6bkPc8+zfR5Df/S6fK/uGq8RL5BXpk/gsdJCIYrkXLnOdbOe463G3IByAKH3ScSEVC9l+zmIjaOG1wXnkzRblcFxpyWJxt4fVJW7tPZpN6WZTs9xTalcJydqqlrnwQlzGgnOOhriPiMJ7Z3BYc9l8ZOywNqYzVsD4L2eCTYX1PIGsA2JHCL61fMQaGgMtmWGbm2zbcFraLWXeyllCWGtGMbt7Xq5c/C5WpUs/S2fnRbFDWyQyCSF5ZINTmnTyHgI4joVt5Q5CU9S8yNJhlOlxjALHn5TmHh4wRfhuomj3LmB15qpz2fJji3snizi51uYJDK2FnC8m1vVm/KzHQTvoOwydxE1FLFM5tnPbdwGrOBINuK4PMue3VYA6iDj2zJWEfWDmkdfUuup4GxsayNoaxoDWtGoACwAVf7reKDNipmnsid9eBwAAtYDykuP1VwcCs/GRzFZZ1+S0vw0c2izU0U3crVERexKAREQBERAEREAREQBERAEREAREQBERAEB2a+LQRxoikFhZObo5a0R1zXPtoEsdi+3ftOvlGniK66HLTD3C4qmjie17XdDmhUesrl1skYeo85Xjy1djv3G6NeSLfxPdEo4wd6Lpn8AY0tZfje8DRyAqscfxuWrlMsx4mNb2kbfktH5nWei0ai34XAUcM7wWne9f8GM6kpawuqyWy2mpAI3jfacamk2kj8B2zvTo2WXKorFajCtHMqK6MYycXdFy0e6DQPAzpHRk8EkT9HOwOHWveXLnD2jukHibFK49TVSaLmPImHb0OXavQ29PIszGd0xgBbRxFzv5kwswcYYDd3PZVzV1L5XukleXyON3OdrJ/3wcC8UV7DYSjh1amte3W+30sjXKcpawiIrJgEREB6VMDo3uZI0te0kOa7WCOBeauvKzJGKsGdfe6gCzZAL3HA14+MOsdRqrGsm6qlJ3+I5n8xnZQnjzhq5HWKoYPKFLEJK9pbvTh38DbUpOHIiUWAVlXzUEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBZAJNgLk6AALkk6gBwlb+E4LUVJtTwueOF1rRDledA5NatDJHIiOlIlmIkqeAge1ReADrd3x5gOGni8dSwy+J3lsS1/x195shTc9RXv9jMQ/uj+lnrWVeCLif1uv/bHv9Sz/AKeO8ws8B5CiLiyLC2lE5Xd1PUMiL3tD8uJzZazKIi2GIREQBERAEREAREQBERAEREAREQBERAEREBhbuD/Dx+EiLCp8jMo6y/KT4NngheqIvBfVLmdHYgiIhB//2Q==",   
```

Comment:
Let's use this short image url instead
```suggestion
	        "https://pbs.twimg.com/profile_images/1548041354806059008/CfEL7qut_400x400.jpg",   
```